internal/commands/image/rm.go

```diff
@@ -14,13 +14,13 @@ import (
 
 func rmCmd() *cobra.Command {
 	return &cobra.Command{
-		Short:   "Remove an application image",
+		Short:   "Remove an App Image",
 		Use:     "rm [APP_IMAGE] [APP_IMAGE...]",
 		Aliases: []string{"remove"},
 		Args:    cli.RequiresMinArgs(1),
 		Example: `$ docker app image rm myapp
 $ docker app image rm myapp:1.0.0
-$ docker app image rm docker.io/library/myapp@sha256:beef...`,
+$ docker app image rm myrepo/myapp@sha256:beef...`,
```

Comment:
nit: we could make the SHA prefix something like `c0de` instead of `beef` given that we're making developer tools

Reply:
done